apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -235,19 +261,14 @@ def backward_pass(trajectory, states_dict, leaf_q_value: torch.Tensor):
         if action_idx is None:  # leaf node
             reward += leaf_q_value
         else:
-            (
-                _,
-                old_idx_to_new_idx,
-                _,
-                N_s_a,
-                q_values,
-                _,
-            ) = states_dict[state]
+            possible_states, repetitions, N_s_a, q_values, _ = states_dict[
+                state
+            ]
```

Comment:
please keep the old code modification (the code in red should replace the code in green)